fal/controllers/load_teams.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+from typing import Sequence, List, TYPE_CHECKING
+import configparser
+import dataclasses
+
+from fal.clients.mfalncfm_main import session_scope
+from fal.models import Team, Anime, Season, TeamWeeklyAnime
+
+if TYPE_CHECKING:
+    from sqlalchemy.orm import Session
+
+
+@dataclasses.dataclass
+class TeamLines():
```

Comment:
What are your thoughts on using a NamedTuple instead of data class? I was looking up the differences and NamedTuples are immutable unlike data classes, so I think that fits our need more. However, it's a pretty trivial issue and it doesn't matter that much either way.

Reply:
If immutability is the primary concern I could go ahead and make it a frozen dataclass. My understanding is that namedtuples are best when you really need to work with a tuple, otherwise dataclasses are the better default. Some discussion here of the difference: https://www.python.org/dev/peps/pep-0557/#why-not-just-use-namedtuple

Reply:
More here: https://www.reddit.com/r/Python/comments/8hf643/namedtuples_vs_dataclasses/

Reply:
Oh, that makes sense. In that case, I think using a frozen dataclass is better.